Update place.php

mjenkins1131 opened the pull request:
Possible solution for placing order with an empty cart.

mjenkins1131 committed:
Update place.php

mjenkins1131 committed:
Update place.php

mitchellm reviewed (changes requested):
The issue is that this doesn't actually prevent the checkout code from firing, the conditional should be & with the "if ($session->isLoggedIn()) {" conditional

mjenkins1131 committed:
Update place.php

Noticed I made a mistake. It was "$totalItems < 1" instead of "$totalItems > 0".
